tests/test_market.py

```diff
@@ -132,6 +133,59 @@ def market_matcher():
     return PayAsBidMatcher()
 
 
+class TestTwoSidedMarket:
+    """Class Responsible for testing two sided market's functionality.
+
+    TODO: bring all related two sided market tests from this module into this class.
```

Comment:
Is this a TODO for this PR?

Reply:
Nope I did not plan to move these tests into the class in the context of this PR, however I'll give it another look and may move them if the won't take long time